airflow/providers/oracle/operators/oracle.py

```diff
@@ -93,4 +95,10 @@ def __init__(
     def execute(self, context: Context):
         self.log.info("Executing: %s", self.procedure)
         hook = OracleHook(oracle_conn_id=self.oracle_conn_id)
-        return hook.callproc(self.procedure, autocommit=True, parameters=self.parameters)
+        ti=context.get('task_instance')
+        try:
+            return hook.callproc(self.procedure, autocommit=True, parameters=self.parameters)
+        except oracledb.DatabaseError as e:
+            code, mesg = e.args[0].message[:-1].split(': ', 1)
+            ti.xcom_push(key='ORA', value=str(code.split('-')[1]))
+            raise AirflowException(f"Task failed with exit code: {code}")
```

Comment:
Instead of coercing the exception to AirflowException, I would prefer this to simply re-raise the original exception:

```suggestion
            raise
```

Reply:
@uranusjr  applied. Also removed import. thanks.